ViMac-Swift/Modes/HintsViewController.swift

```diff
@@ -71,19 +71,19 @@ class HintsViewController: NSViewController {
         let view = HintView(associatedElement: hint.element, hintTextSize: CGFloat(textSize), hintText: hint.text, typedHintText: "")
         guard let elementFrame = self.elementFrame(hint.element) else { return nil }
         
-        var elementCenter: NSPoint
-        if hint.element.role == "AXLink" {
-            // It's likely that the hint text size chosen by the user is roughly equivalent to their viewing text size, hence textSize being used as the offset.
-            elementCenter = GeometryUtils.corner(elementFrame, top: false, right: false, offset: CGFloat(textSize))
-        }
-        else {
-            elementCenter = GeometryUtils.center(elementFrame)
-        }
-
-        let hintOrigin = NSPoint(
-            x: elementCenter.x - (view.intrinsicContentSize.width / 2),
-            y: elementCenter.y - (view.intrinsicContentSize.height / 2)
-        )
+        let hintOrigin: NSPoint = {
+            // position hint on bottom-left of AXLinks (see #373)
+            if hint.element.role == "AXLink" {
+                return elementFrame.origin
+            }
+            
+            // position hint on center of element
+            let elementCenter = GeometryUtils.center(elementFrame)
+            return NSPoint(
+                x: elementCenter.x - (view.intrinsicContentSize.width / 2),
+                y: elementCenter.y - (view.intrinsicContentSize.height / 2)
```

Comment:
@dexterleng I'm just curious about the intrinsic sizes and how they work in relation to the center of an element. From what I've read, the intrinsic size would be something like the size of all the text. 

So if you have the center of an element, what does subtracting half the intrinsic width/height do? I realize that it's moving the hint down and to the left a little. Is this just so the hint obscures the element content less?

Reply:
In macOS, you position views by their origin. The origin is the bottom-left corner of the view you want to render.

`hintOrigin` sets the bottom-left of the HintView. Ideally you would want to set the center coordinate of the HintView to equal `elementCenter`, but you can't do that through Cocoa APIs, so I had to do this math. Does this make sense?

Reply:
Anyway, I really appreciate that you are taking the time to make Vimac better. If you have any ideas or things you would like to see in Vimac, lets talk about it - I'll sure there are little stuff like this that are confusing all over the codebase and I would hate for you to go through all that.

Reply:
Ah, I see. That makes sense. I had misinterpreted the `view` in `view.intrinsicContentSize.width` as the hint element view and not the hint view itself. Thank you for explaining!

And that sounds like loads of fun..😅

Reply:
No problem! Thanks for such a great application. And thank you, I'll take you up on that. I'll try to pick your brain more often!